components/files/__examples__/actions.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import Files from '~/components/files';
+import File from '~/components/files/file';
+import MoreFiles from '~/components/files/more-files';
+import IconSettings from '~/components/icon-settings';
+import Icon from '~/components/icon';
+import Dropdown from '~/components/menu-dropdown';
+
+class Example extends React.Component {
+	static displayName = 'filesLoadingExample';
+
+	render() {
+		return (
+			<IconSettings iconPath="/assets/icons">
+				<Files id="files-with-actions-example">
+					<File
+						id="file-without-title"
+						image="/assets/images/placeholder-img@16x9.jpg"
+						hasNoVisibleTitle
+						labels={{
+							title: 'Proposal.pdf',
+						}}
+						onClickDownload={() => {
+							console.log('Download Button Clicked');
+						}}
+						moreActionsDropdown={
+							<Dropdown
+								id="file-more-actions"
+								iconCategory="utility"
+								iconName="down"
+								iconVariant="bare"
+								onSelect={(value) => {
+									console.log('selected: ', value);
+								}}
+								options={[
+									{ label: 'Menu Item One', value: 'A0' },
+									{ label: 'Menu Item Two', value: 'B0' },
+								]}
+								value="A0"
```

Comment:
For the Dropdown components in this example, please add the following props:
```
align="right"
menuStyle={{ minWidth: '50px', width: '125px' }}
```
This will make the menu dropdown look a little nicer